### DISCUSSION TITLE
Error Loading Keyword Volume Data

### DISCUSSION THREAD

**Discussion by rossdav70** [towfiqi/serpbear#217] [Q&A]
Hi,

I have set up Search Console and Google Ads integration and they appear to be working. I ca see keyword ideas for instance. However when I try to click the "Update Keyword Volume Data" button i get an error message " Error Loading Keyword Volume Data"

Any ideas what I'm doing wrong or if there is a problem?

**Comment by Coysh** [comment]
I'm also getting this issue
